__tests__/meta.test.ts

```diff
@@ -523,6 +555,41 @@ describe('push tag', () => {
         "org.opencontainers.image.licenses=MIT"
       ]
     ],
+    [
+      'event_tag_v2.0.8-beta.67.env',
+      {
+        images: ['org/app', 'ghcr.io/user/app'],
+        tagMatch: `v\\d{1,3}.\\d{1,3}.\\d{1,3}-(alpha|beta).\\d{1,3}`,
+        fullSemver: true,
+        semverPrefix: 'v',
+      } as Inputs,
+      {
+        version: 'v2.0.8-beta.67',
+        latest: true,
+      } as Version,
+      [
+        'org/app:v2.0.8-beta.67',
+        'org/app:v2',
+        'org/app:v2.0',
+        'org/app:v2.0.8',
+        'org/app:latest',
+        'ghcr.io/user/app:v2.0.8-beta.67',
+        'ghcr.io/user/app:v2',
+        'ghcr.io/user/app:v2.0',
+        'ghcr.io/user/app:v2.0.8',
```

Comment:
`v2`, `v2.0` and `v2.0.8` tags should not be generated for pre-release. As explained in [semver specification](https://semver.org/) prerelease versions are updated frequently, and contain many breaking changes that are (by the author's design) not yet fit for public consumption.